packages/scrum-client/src/constants/commonFunctions.js

```diff
@@ -0,0 +1,14 @@
+const getValueFromLocalStorage = (itemId, key) => {
+  if (localStorage && itemId && key) {
```

Comment:
there is a util package for functions , the folder constants is for constants only